BentoKit/BentoKit/Components/TextInput.swift

```diff
@@ -9,6 +9,17 @@ extension Component {
         public let styleSheet: StyleSheet
         public let focusEligibility: FocusEligibility
 
+        /// Creates Component.TextInput
+        /// - parameter title: Label of a the text input which describes it (i.e: "E-mail").
+        /// - parameter placeholder: Placeholder which is displayed instead of text when text is empty.
+        /// - parameter text: Prefilled value of the TextInput. It can be modifed by user. (i.e: "email@email.com"). Limited to only one line.
```

Comment:
```suggestion
        /// - parameter text: Pre-filled value of the TextInput. It can be modifed by user. (i.e: "email@email.com"). Limited to only one line.
```